examples/vm_scheduling/offline_lp/vm_scheduling_ilp.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+import math
+import numpy as np
+from pulp import GLPK, LpInteger, LpMaximize, LpProblem, LpStatus, LpVariable, lpSum
+
+from maro.utils import DottableDict
+
+from common import IlpAllocatedVmInfo, IlpFutureVmInfo, IlpPmCapacity, IlpVmInfo
+
+
+# To indicates not or can not allocate any PM for current VM request.
```

Comment:
can or can not?

Reply:
to indicate two situations: decide to not allocate now or cannot allocate now.


change to 
`To indicate the decision of not allocate or cannot allocate any PM for current VM request` ?

Reply:
Later one sounds better.